src/Libraries/Microsoft.Extensions.AI.Evaluation.NLP/F1EvaluatorContext.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma warning disable S3604
+// S3604: Member initializer values should not be redundant.
+// We disable this warning because it is a false positive arising from the analyzer's lack of support for C#'s primary
+// constructor syntax.
+
+namespace Microsoft.Extensions.AI.Evaluation.NLP;
+
+/// <summary>
+/// Contextual information that the <see cref="F1Evaluator"/> uses to compute the F1 score for a response.
+/// </summary>
+/// <remarks>
+/// <see cref="F1Evaluator"/> measures the F1 score of a response compared to a reference. F1 (Google-BLEU)
+/// is a metric used to evaluate the quality of machine-generated text.
+/// </remarks>
+public sealed class F1EvaluatorContext : EvaluationContext
+{
+    /// <summary>
+    /// Gets the unique <see cref="EvaluationContext.Name"/> that is used for
+    /// <see cref="F1EvaluatorContext"/>.
+    /// </summary>
+    public static string F1ContextName => "F1 Context";
```

Comment:
```suggestion
    public static string GroundTruthContextName => "Ground Truth (F1)";
```

This would match the naming convention we use for GroundTruth in other contexts like [CompletenessEvaluatorContext ](https://github.com/dotnet/extensions/blob/main/src/Libraries/Microsoft.Extensions.AI.Evaluation.Quality/CompletenessEvaluatorContext.cs#L30), [GroundednessProEvaluatorContext](https://github.com/dotnet/extensions/blob/main/src/Libraries/Microsoft.Extensions.AI.Evaluation.Safety/GroundednessProEvaluatorContext.cs#L29) etc.

Reply:
The name is mainly used only for reporting as a label for "contents" passed to the base constructor.

If there is more than one property, we'll need to come up with a name that encompasses all of them. The name of the constant itself does not matter as much since it is not directly used anywhere else - however, since it is public, ideally the name can call out what the context is and can be distinct from the name of the class itself (which is mainly what we were trying to ensure with the existing convention).